app/src/main/res/layout/story_fragment.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.story.StoryViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+      android:id="@+id/story_progress_chapter_completed_text"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_margin="8dp"
+      android:fontFamily="sans-serif-medium"
+      android:textColor="@color/black"
+      android:textSize="14sp"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.recyclerview.widget.RecyclerView
```

Comment:
Does this need to be in a scrollview? Also, it seems per https://xd.adobe.com/view/bee1fced-33bd-4f69-6a89-2c30fe1c4f65-c8d0/screen/5abb0ace-19ee-48b8-8154-919154f924d2/Home-Page-1-Story-Pre-Scroll-1/ that the header should scroll with the contents. Does that mean that the header needs to be part of the chapter list recycler view as a separate view type?

Reply:
Done, added as a part of recyclerView